backup-loop.sh

```diff
@@ -59,9 +81,9 @@ while true; do
     log ERROR "rcon save-off command failed"
   fi
 
-  if (( ${PRUNE_BACKUPS_DAYS} > 0 )); then
+  if (( PRUNE_BACKUPS_DAYS > 0 )); then
```

Comment:
Interesting, it seems to accept with or without `$`-ness

Reply:
Yes. And using it sometimes can lead to subtle errors. See: https://github.com/koalaman/shellcheck/wiki/SC2004
I tend to write bash with shellcheck linting / analyzer, cause it makes it more robust, at least in my opinion.